### ISSUE TITLE
Error while communicating with SysAp

### ISSUE THREAD

**Issue by kochmilch** [henry-spanka/homebridge-buschjaeger#22]
hello. i was very happy about the possibility to control our free@home system with Siri. As a newbee i chose the Hoobs, Homebridge Out Of the Box. 

Unfortunately i always get the following errors:
8.5.2020, 18:58:29 [BuschJaegerSysAp] Trying to connect to SysAP
8.5.2020, 18:58:29 [BuschJaegerSysAp] Error while communicating with SysAp

Think it is related to the configuration of sysIP. Where can I find the right one? The IP address under the network settings does not seem to be. 

{
    "platform": "BuschJaegerSysAp",
    "sysIP": "192.168.2.107",
    "updateInterval": 60,
    "mappings": {}
}

It's a bit embarrassing. Can someone please give me a hand? 
Thank you very much.

**Comment by henry-spanka** [comment]
Hi,

tbh the documentation is a little bit misleading. It's not actually the IP of the SysAp. The SysIP needs to be set to the IP where the [freeathome-api](https://github.com/henry-spanka/freeathome-api) is running. The plugin won't work without it.

It's not really Out of the Box. You first need to install the freeathome-api (preferably on Hoobs and set the SysIP to "localhost". The plugin won't work without it.

So basically you need SSH access to your Hoobs system and then follow the instructions for freeathome-api first and set SysIP to "localhost".

**Comment by kochmilch** [comment]
I'm lost. just trying to get really deep into this subject. And actually, these things are a little bit my cup of tea. Apparently not enough. In the end I seem to fail with the correct installation of the free@home API. Which is a real pity as we decided on this basis, for Bush und Jäger plus Hoobs. 
Your documentation is certainly sufficiently explanatory. Especially since you also gave the reference to localhost. 
My sincere thanks for the friendly support. Also my thanks for the great work and all the best for the future.

**Comment by henry-spanka** [comment]
Documentation might be good enough for users experienced with homebridge but not so good for newbies. Where exactly are you stuck?

**Comment by kochmilch** [comment]
Feel like a kid holding a rubic cube for the first time. It's just embarrassing. In the end I only get warnings when I install the plugins, don't know which directories to put them in and just don't know what comes next and when. 
I will have to get used to Homebridge and move this project for now. As bitter as that is. 
Thank you for your kind help. Think that if it works, the plugin will be unspeakably helpful.

**Comment by kochmilch** [comment]
hi. I've been running it for about three weeks now without any problems. 
Thanks a lot for this fantastic plugin, Henry. 
In the end it helped to create a new user with the role "Installer" in the Sys AP, who had the same user name as the hoob. Of course it could be that I had to hang up a dream catcher at full moon but somehow it helped. Additionally, it was apparently important to upgrade to the last stable version of npm (_npm install_ -g npm respectively _sudo n stable_).
Anyway, I can now finally control 16 Free@Home roller shutters with HomeKit using the Hoobs... awesome :D

**Comment by henry-spanka** [comment]
👍 Happy to hear that.
